consoleApp/src/main/java/by/storeApp/StoreApp.java

```diff
@@ -0,0 +1,15 @@
+package by.storeApp;
```

Comment:
if your goupId is by.issoft and artifactId is store - your module base package should be by.issoft.store.****. And please change it in every class/module.

Reply:
updated